packit/api.py

```diff
@@ -403,12 +403,15 @@ def create_srpm(
     def status(self):
         status = Status(self.config, self.package_config, self.up, self.dg)
 
-        ds_prs = status.get_downstream_prs()
-        if ds_prs:
-            logger.info("Downstream PRs:")
-            logger.info(tabulate(ds_prs, headers=["ID", "Title", "URL"]))
-        else:
-            logger.info("Downstream PRs: No open PRs.")
+        try:
+            ds_prs = status.get_downstream_prs()
+            if ds_prs:
+                logger.info("Downstream PRs:")
+                logger.info(tabulate(ds_prs, headers=["ID", "Title", "URL"]))
+            else:
+                logger.info("Downstream PRs: No open PRs.")
+        except Exception as jde:
```

Comment:
Could be some specific Exception used here? e.g. `jsonschema.exceptions.ValidationError`

Reply:
Also the `try:` section is too wide. Do we need all the `logger.info()` in there?